user_tools/docs/user-tools-dataproc.md

```diff
@@ -197,7 +197,7 @@ spark_rapids_user_tools dataproc qualification --help
 
 ```
 spark_rapids_user_tools dataproc profiling [options]
-spark_rapids_user_tools dataproc profiling --help
+spark_rapids_user_tools dataproc profiling -- --help
```

Comment:
Is adding the extra dashes intentional?

Reply:
> Is adding the extra dashes intentional?

yes, because "profiling" has no positional arguments. The python CLI won't be able to detect if help is an actual argument to the cm or is a flag